package.json

```diff
@@ -15,7 +14,8 @@
     "url": "git+https://github.com/pusher/chatkit-client-js.git"
   },
   "dependencies": {
-    "pusher-platform": "^0.15.0"
+    "pusher-platform": "^0.15.0",
+    "ramda": "^0.25.0"
```

Comment:
This can still just be in `devDependencies` can it not?

Reply:
nah it's used everywhere. It was in the wrong place before.